src/main/java/com/odysseusinc/athena/api/v1/controller/converter/ConceptSearchPhraseToSolrQueryConverter.java

```diff
@@ -0,0 +1,172 @@
+package com.odysseusinc.athena.api.v1.controller.converter;
+
+import com.odysseusinc.athena.api.v1.controller.dto.ConceptSearchDTO;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import org.apache.commons.collections.CollectionUtils;
+import org.apache.commons.lang.StringUtils;
+import org.apache.solr.client.solrj.util.ClientUtils;
+
+public class ConceptSearchPhraseToSolrQueryConverter {
```

Comment:
Probably it's better not to call this `...Converter` since not implemented `Converter` interface and may confuse.